src/backend/marsha/core/serializers.py

```diff
@@ -477,11 +478,15 @@ def get_url(self, obj):
         if obj.uploaded_on is None:
             return None
 
-        url = "{protocol:s}://{cloudfront:s}/{pk!s}/document/{stamp:s}".format(
+        url = (
+            "{protocol:s}://{cloudfront:s}/{pk!s}/document/{stamp:s}"
+            "?response-content-disposition={content_disposition:s}"
+        ).format(
             protocol=settings.AWS_S3_URL_PROTOCOL,
             cloudfront=settings.CLOUDFRONT_DOMAIN,
             pk=obj.pk,
             stamp=time_utils.to_timestamp(obj.uploaded_on),
+            content_disposition=quote_plus("attachment; filename=" + obj.title),
```

Comment:
Looking at below's test, it seems that the `attachment; filename=` part can be added to the above string and only the object title needs to be passed to `quote_plus`?

Reply:
you mean added like this `attachment%3B+filename%3D` ?

Reply:
oups sorry forget this comment ;-)